quickwit-cli/src/index.rs

```diff
@@ -637,11 +636,10 @@ pub async fn ingest_docs_cli(args: IngestDocsArgs) -> anyhow::Result<()> {
     if args.overwrite {
         reset_index(&index_metadata, metastore.clone(), storage.clone()).await?;
     }
-    let pipeline_params =
-        IndexingPipelineParams::try_new(index_metadata, indexer_config, metastore, storage).await?;
-    let pipeline = IndexingPipeline::new(pipeline_params);
-    let universe = Universe::new();
-    let (_pipeline_mailbox, pipeline_handle) = universe.spawn_actor(pipeline).spawn_async();
+    let client = IndexingServer::spawn(indexer_config, metastore);
+    client
+        .spawn_pipeline(args.index_id.clone(), Some(source_config))
```

Comment:
In this PR, I'm handling both `quickwit index ingest` and `quickwit service indexer` commands with the same `IndexingServer`. 

Consequently, most indexing server APIs accept an optional source parameter. `None` means we launch an indexing pipeline for each source defined for the target index.

Reply:
Could it be two methods then?